docs/src/pages/demos/lists/VirtualizedList.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import { FixedSizeList } from 'react-window';
```

Comment:
What dependency should we use? https://npm-stat.com/charts.html?package=react-virtualized&package=react-window

Reply:
From the react-window FAQs 
> If react-window provides the functionality your project needs, I would strongly recommend using it instead of react-virtualized.

I'm curious what the others think @mui-org/core-contributors

Reply:
And so we showcase the two dependencies, it's good for me.

Reply:
Do what the author says i.e. use `react-window`.